docker/localstack_init/localstack_init.sh

```
@@ -3,32 +3,57 @@ set -e
     local SERVICE_NAME=$1
     local SECRET_FILE=$2
     local DESCRIPTION=$3
-    
-    # 시크릿 이름 규칙: ticket-{서비스명}/secure-config
+
+    # 시크릿 이름 규칙: {서비스명}-svc/secure-config
     local SECRET_NAME="${SERVICE_NAME}-svc/secure-config"
-    
+
     # Docker Secret 파일에서 실제 비밀번호 읽기
     local DB_PASSWORD=$(cat "/run/secrets/${SECRET_FILE}")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)